backend/mora/common.py

```diff
@@ -374,6 +374,7 @@ def create_organisationsenhed_payload(
     enhedstype: str,
     overordnet: str,
     adresser: typing.List[dict] = None,
+    integrationdata: str = ""
```

Comment:
Since these are two words, it would be better to call this variable 'integration_data`.

Reply:
It seems to me that using Danish would be consistent with the other parameters, so `integrationsdata`.

Reply:
Well, but if it's supposed to be read as English, it should be with an underscore. :-)